client/webserver/site/src/js/markets.js

```diff
@@ -679,6 +679,9 @@ export default class MarketsPage extends BasePage {
           this.showCancel(row, ord.id)
         })
       }
+      const side = Doc.tmplElement(row, 'side')
+      const cssClass = ord.sell ? 'sellcolor' : 'buycolor'
+      side.classList.add(cssClass)
```

Comment:
I'd do a one-liner here.

`side.classList.add(ord.sell ? 'sellcolor' : 'buycolor')`

Reply:
Thanks for the suggestion.